libs/ui/src/diagnostics/components.ts

```diff
@@ -0,0 +1,5 @@
+import styled from 'styled-components';
+
+export const DiagnosticSection = styled.section`
+  margin-top: 1.5em;
```

Comment:
Pretty arbitrary, open to design feedback. I reduced the margins between sections slightly because they felt a little spaced out but could be convinced to increase this.

Reply:
I like the spacing!

I often prefer to use a `gap` style rule on the container to space out sections rather than add margin to the sections, because it makes the section component itself more reusable